### PR TITLE
getfattr, setfattr, sysdig: add page

### LINKED ISSUE #13118: Page request: sysdig
### Command description

he definitive system and process troubleshooting tool

### Command details

_No response_

### Documentation

_No response_

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### LINKED ISSUE #13485: Page request: getfattr, setfattr
### Command description

Commands for extended filesystem attributes. I have no experience with these commands so I didn't dare to make the pages as I don't know what commands are most used.

### Command details

_No response_

### Documentation

https://manned.org/man/getfattr

### Platform

Linux

### VCS repository link (e.g. GitHub, GitLab)

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by angadsgrover**
Resolves #13118
Resolves #13485 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).

### CODE REVIEW

File: pages/linux/sysdig.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+`sysdig -r {filename}.scap`
+
+- Filter and Print all the open system calls invoked by cat:
+
+`sysdig proc.name=cat and evt.type=open`
+
+- Register any found plugin and use dummy as input source passing to it open params:
+
+`sysdig -I dummy:10'`
```
[the remark this answers is not in the document]
Comment (angadsgrover) [reply pages/linux/sysdig.md:25]: The 10 can be an assigned parameter by the user. Corrected it in the commit along with other recommended changes.

**Comment by angadsgrover** [comment]
@sebastiaanspeck 
I apologize for the confusion. I was trying to create another PR through a separate branch for another issue but accidentally made a mistake with the branches. Would you recommend that I create a new PR for both changes by deleting this PR?

**Comment by angadsgrover** [comment]
@sebastiaanspeck 
I apologize for the confusion. I was trying to create another PR through a separate branch for the issue but accidentally made a mistake with the branches. 
I have since corrected the issues in both files and have pushed the changes

File: pages/linux/getfattr.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# getfattr
+
+> Display file names and extended attributes.
+> More information: <https://manned.org/man/getfattr>.
+
+- To retrieve all extended attributes of a file and display them in a detailed format:
```
Comment (spageektti) [inline pages/linux/getfattr.md:6]:
```suggestion
- Retrieve all extended attributes of a file and display them in a detailed format:
```
See style guide.

File: pages/linux/getfattr.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+`getfattr -d {{path/to/file}}`
+
+- To get a specific attribute of a file:
```
Comment (spageektti) [inline pages/linux/getfattr.md:10]:
```suggestion
- Get a specific attribute of a file:
```

File: pages/linux/setfattr.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# setfattr
+
+> Set extended file attributes.
+> More information: <https://manned.org/man/setfattr>.
+
+- To set name of attribute for file:
```
Comment (spageektti) [inline pages/linux/setfattr.md:6]:
```suggestion
- Set name of attribute for file:
```

File: pages/linux/setfattr.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+`setfattr -n user.{{attribute_name}} {{path/to/file}}`
+
+- To set a user-defined value of an extended attribute on a file:
```
Comment (spageektti) [inline pages/linux/setfattr.md:10]:
```suggestion
- Set a user-defined value of an extended attribute on a file:
```

File: pages/linux/setfattr.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+`setfattr -n user.{{attribute}} -v "{{value}}" {{path/to/file}}`
+
+- To remove a specific attribute of a file:
```
Comment (spageektti) [inline pages/linux/setfattr.md:14]:
```suggestion
- Remove a specific attribute of a file:
```

**Pull request merged**